<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_pulse</class>
 <widget class="QDialog" name="Dialog_pulse">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="comboBox_3">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>11</y>
     <width>331</width>
     <height>21</height>
    </rect>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="duplicatesEnabled">
    <bool>false</bool>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <item>
    <property name="text">
     <string>碳钢 80%氩气</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>不锈钢 80%氩气</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>不锈钢 97.5%氩气</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>铜硅合金 100%氩气</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>铝硅合金 100%氩气</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>铝镁合金</string>
    </property>
   </item>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>461</width>
     <height>171</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Microsoft YaHei UI'; font-size:16pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;send_data&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>104</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>脉冲</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>441</width>
     <height>231</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_11">
      <property name="lineWidth">
       <number>3</number>
      </property>
      <property name="text">
       <string>电压微调</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_5">
      <property name="lineWidth">
       <number>3</number>
      </property>
      <property name="text">
       <string>电压 V</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="label_2">
      <property name="minimumSize">
       <size>
        <width>24</width>
        <height>21</height>
       </size>
      </property>
      <property name="lineWidth">
       <number>3</number>
      </property>
      <property name="text">
       <string>电感</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QSpinBox" name="spinBox">
      <property name="minimum">
       <number>-10</number>
      </property>
      <property name="maximum">
       <number>10</number>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QDoubleSpinBox" name="doubleSpinBox">
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="minimum">
       <double>-5.000000000000000</double>
      </property>
      <property name="maximum">
       <double>5.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLabel" name="label_6">
      <property name="minimumSize">
       <size>
        <width>24</width>
        <height>21</height>
       </size>
      </property>
      <property name="lineWidth">
       <number>3</number>
      </property>
      <property name="text">
       <string>丝径</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QComboBox" name="comboBox">
      <item>
       <property name="text">
        <string>1.0</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>0.8</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>0.9</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLabel" name="label_8">
      <property name="minimumSize">
       <size>
        <width>33</width>
        <height>21</height>
       </size>
      </property>
      <property name="lineWidth">
       <number>3</number>
      </property>
      <property name="text">
       <string>2T/4T</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QComboBox" name="comboBox_2">
      <item>
       <property name="text">
        <string>2T</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4T</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_12">
      <property name="lineWidth">
       <number>3</number>
      </property>
      <property name="text">
       <string>速度m/min</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="label_7">
      <property name="lineWidth">
       <number>3</number>
      </property>
      <property name="text">
       <string>电流 A</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QLabel" name="label_4">
      <property name="minimumSize">
       <size>
        <width>48</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>48</width>
        <height>21</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <italic>false</italic>
        <bold>false</bold>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="lineWidth">
       <number>3</number>
      </property>
      <property name="text">
       <string>后吹时间</string>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="minimum">
       <double>0.100000000000000</double>
      </property>
      <property name="maximum">
       <double>2.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QDoubleSpinBox" name="doubleSpinBox_3">
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="minimum">
       <double>1.000000000000000</double>
      </property>
      <property name="maximum">
       <double>12.800000000000001</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
      <property name="value">
       <double>10.000000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
